extensions/binding/org.eclipse.smarthome.binding.astro/OSGI-INF/AstroDiscoveryService.xml

```diff
@@ -9,7 +9,7 @@
     http://www.eclipse.org/legal/epl-v10.html
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.smarthome.binding.astro.discovery">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="false" name="org.eclipse.smarthome.binding.astro.discovery">
```

Comment:
The implementation of `DiscoveryServiceRegistry` (`DiscoveryServiceRegistryImpl`) is the consumer of all `DiscoveryService`s.